boards/ws_esp32_s3_matrix.json

```diff
@@ -0,0 +1,32 @@
+{
+  "build": {
+    "core": "esp32",
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "memory_type": "qio_qspi"
```

Comment:
This needs a comma, it being missing is causing issues: https://github.com/platformio/platformio-core/issues/5055